Modeling rules for Taniumm Threat Response. (#26230)
demisto/content: Packs/TaniumThreatResponse/README.md

# Tanium Threat Response
This pack includes Cortex XSIAM content.

## Configuration on Server Side

> You need to configure a Socket Receiver on Tanium side. 

Reviewer: ```suggestion
You need to configure a Socket Receiver on the Tanium side. 
```

> Follow the below steps to configure the socket receiver:

Reviewer: ```suggestion
Perform the following steps to configure the Socket Receiver:
```

1. Go to **Modules** > **Connect**.
2. Enter a name and description for the connection.

> 3. Select **Event** from the dropdown under **Source**.

Reviewer: ```suggestion
3. From the **Source** dropdown, select **Event**.
```

> 3. Select **Event** from the dropdown under **Source**.

Reviewer: ```suggestion
3. From the **Source** dropdown, select **Event**.
```

> 4. Select **Tanium Threat Response** from the dropdown under **Event Group**.

Reviewer: ```suggestion
4. From the **Event Group** dropdown, select **Tanium Threat Response**.
```

5. Select **Match Alerts Raw**.

> 6. Select **Socket Receiver** from the dropdown under **Destination**.

Reviewer: ```suggestion
6. From the **Destination** dropdown, select **Socket Receiver**.
```

7. Specify a unique name for the **Destination Name**.

> 8. Under **Host** fill in the name or the ip address of the siem.

Reviewer: ```suggestion
8. Under **Host**, fill in the name or the IP address of the SIEM.
```
